src/crypto/hash/index.js

```diff
@@ -143,18 +144,18 @@ export default {
    */
   getHashByteLength: function(algo) {
     switch (algo) {
-      case 1: // - MD5 [HAC]
+      case enums.hash.md5: // - MD5 [HAC]
```

Comment:
Could you update the above `digest` function in the same way? Also, I think these comments aren't really necessary anymore, now.
```suggestion
      case enums.hash.md5:
```